pkg/remote/codec/util.go

```diff
@@ -49,6 +49,9 @@ func SetOrCheckMethodName(methodName string, message remote.Message) error {
 	if err != nil {
 		return err
 	}
+	if codec := svcInfo.GenericCodec; codec != nil {
+		codec.SetMethod(methodName)
```

Comment:
`codec` looks like a global variable, so set local variable methodName to it is not correct.

Reply:
oh...